pyproject.toml

```diff
@@ -22,6 +22,8 @@ docker = "^4.3.1"
 bech32 = "^1.1.0"
 multitail2 = "^1.5.2"
 python-dotenv = "^0.19.2"
+pyyaml-include = "^1.3"
+mergedeep = {git = "https://github.com/mmsqe/mergedeep.git", rev = "nested-merge"}
```

Comment:
we already used a `jsonmerge`, what's the difference with this one?

Reply:
just more strategies, but can use without nested merge